watchdog/handler.go

```diff
@@ -228,7 +228,9 @@ func getAdditionalEnvs(config *WatchdogConfig, r *http.Request, method string) [
 		}
 
 		envs = append(envs, fmt.Sprintf("Http_Method=%s", method))
+		// DEPRECATED: Http_ContentLength will be deprecated in favour
```

Comment:
Let's put instead:

```
// Deprecation notice: 
```

Could you add a unit test that shows/proves the new value is added and accessible in a function?

Reply:
Will do :+1: